From 152304f3fd2ebd04ca6d49c62dce4ddd2269326a Mon Sep 17 00:00:00 2001
From: Frederic Crozat <fcrozat@suse.com>
Date: Fri, 20 Jul 2012 11:07:29 +0200
Subject: [PATCH] Revert "libply: remove the unused SPLASH_MODE_SUSPEND and
 SPLASH_MODE_RESUME"

This reverts commit 02c596757753589f7e49d26eab2759d552f24671.

Conflicts:
	src/libply-splash-core/ply-boot-splash-plugin.h
	src/plugins/splash/script/script-lib-plymouth.c
---
 src/libply-splash-core/ply-boot-splash-plugin.h |    2 ++
 src/plugins/splash/script/script-lib-plymouth.c |    6 ++++++
 2 files changed, 8 insertions(+)

--- plymouth-0.9.2/src/libply-splash-core/ply-boot-splash-plugin.h.orig	2015-05-24 21:36:57.996946648 +0200
+++ plymouth-0.9.2/src/libply-splash-core/ply-boot-splash-plugin.h	2015-05-24 21:40:09.206938626 +0200
@@ -38,6 +38,8 @@
         PLY_BOOT_SPLASH_MODE_BOOT_UP,
         PLY_BOOT_SPLASH_MODE_SHUTDOWN,
         PLY_BOOT_SPLASH_MODE_REBOOT,
+        PLY_BOOT_SPLASH_MODE_SUSPEND,
+        PLY_BOOT_SPLASH_MODE_RESUME,
         PLY_BOOT_SPLASH_MODE_UPDATES,
         PLY_BOOT_SPLASH_MODE_SYSTEM_UPGRADE,
         PLY_BOOT_SPLASH_MODE_FIRMWARE_UPGRADE,
--- plymouth-22.02.122/src/plugins/splash/script/script-lib-plymouth.c.orig	2022-08-27 20:59:50.679508419 +0200
+++ plymouth-22.02.122/src/plugins/splash/script/script-lib-plymouth.c	2022-08-27 21:01:12.459065382 +0200
@@ -73,6 +73,12 @@ static script_return_t plymouth_get_mode
         case PLY_BOOT_SPLASH_MODE_REBOOT:
                 obj = script_obj_new_string ("reboot");
                 break;
+        case PLY_BOOT_SPLASH_MODE_SUSPEND:
+                obj = script_obj_new_string ("suspend");
+                break;
+        case PLY_BOOT_SPLASH_MODE_RESUME:
+                obj = script_obj_new_string ("resume");
+	        break;
         case PLY_BOOT_SPLASH_MODE_UPDATES:
                 obj = script_obj_new_string ("updates");
                 break;
